acceptance/lib/pxp-agent/test_helper.rb

```diff
@@ -6,6 +6,9 @@
 
 # This file contains general test helper methods for pxp-agent acceptance tests
 
+# The standard path for git checkouts is where pcp-broker will be
+GIT_CLONE_FOLDER = '/opt/puppet-git-repos'
```

Comment:
I was able to answer my own question.  This is the place where it's now being standardised, so it can be used in this helper and also in acceptance/setup/common/010_Setup_Broker.rb